packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view_controller.dart

```diff
@@ -40,10 +28,30 @@ class ObjectInspectorViewController extends DisposableController
   ValueListenable<bool> get refreshing => _refreshing;
   final _refreshing = ValueNotifier<bool>(false);
 
+  ValueListenable<bool> get initialized => _initialized;
+  final _initialized = ValueNotifier<bool>(false);
```

Comment:
we aren't listening to this anywhere, so we can just use a private bool _initialzed instead of the notifier / listenable combo

Reply:
done